### ISSUE TITLE
Click on the LU error notification does not show the LU error content 

### ISSUE THREAD

**Issue by bobokids** [microsoft/BotFramework-Composer#2533]
## Describe the bug

. Add a line of LU data with error to the LU trigger phrase
. Notice that an error icon showing besides the "Start Bot" button
. Click the button to get the detail LU error notifications
. Click each error notification
. Composer jump to the Lu All-up view

Expect:
Composer shows the LU data with error I added.

Result:
the error data didn't show up in the view.

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

## Screenshots

. error data added
![image](https://user-images.githubusercontent.com/19182479/78634148-e3985580-78d5-11ea-9ac6-9bc9a59bf3d5.png)
. error indicator
![image](https://user-images.githubusercontent.com/19182479/78634175-f27f0800-78d5-11ea-8439-60295758e9ef.png)
. error notifications
![image](https://user-images.githubusercontent.com/19182479/78634189-fd399d00-78d5-11ea-864a-457b18959215.png)
. when click the error notification item, composer jump to all-up view, and all data are correct there.
![image](https://user-images.githubusercontent.com/19182479/78634271-2eb26880-78d6-11ea-9f83-231b3c24781a.png)

**Comment by boydc2014** [comment]
We've seen some back and force on the notification page behavior, let's meet to clarity the design and solve any potential design issues. @liweitian  @lei9444 @zhixzhan

**Comment by zhixzhan** [comment]
I committed a fix, now error can navigate to all up view editor. 

ps:  
I remember previous a logic is if error happen on a dialog used intent, click will navigate to form editor. but now mostly will not, almost all error navigate to all up view. 
After some debug, I found the reason is luParser can't parse every error intent, meanwhile our find error intent in dialogPath rely on parse results. That caused failed to calculate dialogPath, then failed to navigate to form editor.
